From d07daf9896ba71e602ff3bd2882c4daad9873205 Mon Sep 17 00:00:00 2001
From: Patrick Rudolph <patrick.rudolph@9elements.com>
Date: Wed, 30 Aug 2023 13:21:59 +0200
Subject: [PATCH 2/8] sbp1: Update PID settings

Change settings as requested by IBM.

- Update integral coefficient: -9.2 -> -4.6
- Update negative hysteresis: 5 -> 2
- Update setpoints to higher limits for all groups

Change-Id: I9b0d3cdc1c538b2d534225b3c5224e846a17cb30
Signed-off-by: Patrick Rudolph <patrick.rudolph@9elements.com>
---
 configurations/sbp1_chassis.json | 54 ++++++++++++++++----------------
 1 file changed, 27 insertions(+), 27 deletions(-)

diff --git a/configurations/sbp1_chassis.json b/configurations/sbp1_chassis.json
index 83fe374..54318c0 100644
--- a/configurations/sbp1_chassis.json
+++ b/configurations/sbp1_chassis.json
@@ -1564,7 +1564,7 @@
             "Class": "temp",
             "FFGainCoefficient": 0,
             "FFOffCoefficient": 0,
-            "ICoefficient": -9.2,
+            "ICoefficient": -4.6,
             "ILimitMax": 32000,
             "ILimitMin": 2500,
             "Inputs": [
@@ -1574,13 +1574,13 @@
                 "BCM3_TEMP Temperature"
             ],
             "Name": "PSU Temperature",
-            "NegativeHysteresis": 5,
+            "NegativeHysteresis": 0,
             "OutLimitMax": 32000,
             "OutLimitMin": 2500,
             "Outputs": [],
             "PCoefficient": -536,
             "PositiveHysteresis": 0,
-            "SetPoint": 55,
+            "SetPoint": 85,
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
@@ -1592,7 +1592,7 @@
             "Class": "temp",
             "FFGainCoefficient": 0,
             "FFOffCoefficient": 0,
-            "ICoefficient": -9.2,
+            "ICoefficient": -4.6,
             "ILimitMax": 32000,
             "ILimitMin": 2500,
             "Inputs": [
@@ -1602,13 +1602,13 @@
                 "QSFP4 Temperature"
             ],
             "Name": "QSFP Temperature",
-            "NegativeHysteresis": 5,
+            "NegativeHysteresis": 0,
             "OutLimitMax": 32000,
             "OutLimitMin": 2500,
             "Outputs": [],
             "PCoefficient": -536,
             "PositiveHysteresis": 0,
-            "SetPoint": 55,
+            "SetPoint": 60,
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
@@ -1620,7 +1620,7 @@
             "Class": "temp",
             "FFGainCoefficient": 0,
             "FFOffCoefficient": 0,
-            "ICoefficient": -9.2,
+            "ICoefficient": -4.6,
             "ILimitMax": 32000,
             "ILimitMin": 2500,
             "Inputs": [
@@ -1630,13 +1630,13 @@
                 "Die_CPU4"
             ],
             "Name": "CPU Temperature",
-            "NegativeHysteresis": 2,
+            "NegativeHysteresis": 0,
             "OutLimitMax": 32000,
             "OutLimitMin": 2500,
             "Outputs": [],
             "PCoefficient": -536,
             "PositiveHysteresis": 0,
-            "SetPoint": 65,
+            "SetPoint": 85,
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
@@ -1648,7 +1648,7 @@
             "Class": "temp",
             "FFGainCoefficient": 0,
             "FFOffCoefficient": 0,
-            "ICoefficient": -9.2,
+            "ICoefficient": -4.6,
             "ILimitMax": 32000,
             "ILimitMin": 2500,
             "Inputs": [
@@ -1674,13 +1674,13 @@
                 "PVPP_HBM_CPU3 Temperature"
             ],
             "Name": "CPU VR Temperature",
-            "NegativeHysteresis": 5,
+            "NegativeHysteresis": 0,
             "OutLimitMax": 32000,
             "OutLimitMin": 2500,
             "Outputs": [],
             "PCoefficient": -536,
             "PositiveHysteresis": 0,
-            "SetPoint": 55,
+            "SetPoint": 85,
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
@@ -1692,7 +1692,7 @@
             "Class": "temp",
             "FFGainCoefficient": 0,
             "FFOffCoefficient": 0,
-            "ICoefficient": -9.2,
+            "ICoefficient": -4.6,
             "ILimitMax": 32000,
             "ILimitMin": 2500,
             "Inputs": [
@@ -1703,13 +1703,13 @@
                 "PVCORE_NIC2 Temperature"
             ],
             "Name": "NIC VR Temperature",
-            "NegativeHysteresis": 5,
+            "NegativeHysteresis": 0,
             "OutLimitMax": 32000,
             "OutLimitMin": 2500,
             "Outputs": [],
             "PCoefficient": -536,
             "PositiveHysteresis": 0,
-            "SetPoint": 55,
+            "SetPoint": 60,
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
@@ -1721,7 +1721,7 @@
             "Class": "temp",
             "FFGainCoefficient": 0,
             "FFOffCoefficient": 0,
-            "ICoefficient": -9.2,
+            "ICoefficient": -4.6,
             "ILimitMax": 32000,
             "ILimitMin": 2500,
             "Inputs": [
@@ -1743,13 +1743,13 @@
                 "DIMM_H2_CPU1"
             ],
             "Name": "CPU1 VR Temperature",
-            "NegativeHysteresis": 5,
+            "NegativeHysteresis": 0,
             "OutLimitMax": 32000,
             "OutLimitMin": 2500,
             "Outputs": [],
             "PCoefficient": -536,
             "PositiveHysteresis": 0,
-            "SetPoint": 55,
+            "SetPoint": 60,
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
@@ -1761,7 +1761,7 @@
             "Class": "temp",
             "FFGainCoefficient": 0,
             "FFOffCoefficient": 0,
-            "ICoefficient": -9.2,
+            "ICoefficient": -4.6,
             "ILimitMax": 32000,
             "ILimitMin": 2500,
             "Inputs": [
@@ -1783,13 +1783,13 @@
                 "DIMM_H2_CPU2"
             ],
             "Name": "CPU2 VR Temperature",
-            "NegativeHysteresis": 5,
+            "NegativeHysteresis": 0,
             "OutLimitMax": 32000,
             "OutLimitMin": 2500,
             "Outputs": [],
             "PCoefficient": -536,
             "PositiveHysteresis": 0,
-            "SetPoint": 55,
+            "SetPoint": 60,
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
@@ -1801,7 +1801,7 @@
             "Class": "temp",
             "FFGainCoefficient": 0,
             "FFOffCoefficient": 0,
-            "ICoefficient": -9.2,
+            "ICoefficient": -4.6,
             "ILimitMax": 32000,
             "ILimitMin": 2500,
             "Inputs": [
@@ -1823,13 +1823,13 @@
                 "DIMM_H2_CPU3"
             ],
             "Name": "CPU3 VR Temperature",
-            "NegativeHysteresis": 5,
+            "NegativeHysteresis": 0,
             "OutLimitMax": 32000,
             "OutLimitMin": 2500,
             "Outputs": [],
             "PCoefficient": -536,
             "PositiveHysteresis": 0,
-            "SetPoint": 55,
+            "SetPoint": 60,
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
@@ -1841,7 +1841,7 @@
             "Class": "temp",
             "FFGainCoefficient": 0,
             "FFOffCoefficient": 0,
-            "ICoefficient": -9.2,
+            "ICoefficient": -4.6,
             "ILimitMax": 32000,
             "ILimitMin": 2500,
             "Inputs": [
@@ -1863,13 +1863,13 @@
                 "DIMM_H2_CPU4"
             ],
             "Name": "CPU4 VR Temperature",
-            "NegativeHysteresis": 5,
+            "NegativeHysteresis": 0,
             "OutLimitMax": 32000,
             "OutLimitMin": 2500,
             "Outputs": [],
             "PCoefficient": -536,
             "PositiveHysteresis": 0,
-            "SetPoint": 55,
+            "SetPoint": 60,
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
-- 
2.41.0

